docker_test.go

```diff
@@ -575,6 +578,9 @@ func TestMainFlow(t *testing.T) {
 
 func setupMainFlow(svcCtx context.Context, t *testing.T) <-chan struct{} {
 	setupStart := time.Now()
+	if testing.Verbose() {
+		t.Setenv("LOG_LEVEL", "DEBUG")
```

Comment:
Do we still need this Log level?

Reply:
I was thinking to keep it because these tests fail quite often on CI and it would be useful to have more info. The test length time seems to be the same. WDYT?